<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMImagePropertyEditor</class>
 <widget class="QFrame" name="AMImagePropertyEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AMImagePropertyEditor {
	background-color: rgba(0, 0, 0, 179);
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QRadioButton" name="useStandardColorMapButton">
     <property name="styleSheet">
      <string notr="true">color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>Default Color Map</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="standardColorMapComboBox">
     <property name="currentIndex">
      <number>7</number>
     </property>
     <item>
      <property name="text">
       <string>Autumn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bone</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cool</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Copper</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gray</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HSV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pink</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spring</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Summer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>White</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Winter</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QRadioButton" name="useCustomColorsButton">
     <property name="styleSheet">
      <string notr="true">color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>Custom Colors</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>18</number>
     </property>
     <item>
      <widget class="AMColorPickerButton" name="firstColorButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AMColorPickerButton" name="secondColorButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rgbButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(230,230,230);</string>
         </property>
         <property name="text">
          <string>RGB</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="hsvButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(230,230,230);</string>
         </property>
         <property name="text">
          <string>HSV</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0" rowspan="2" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="brightnessText">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(230,230,230);</string>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelBrightness">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../icons/icons.qrc">:/22x22/brightness.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="5" column="2" rowspan="2">
    <widget class="QSlider" name="brightnessSlider">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelContrast">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../icons/icons.qrc">:/22x22/contrast.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QSlider" name="contrastSlider">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="contrastText">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(230,230,230);</string>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelGamma">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../icons/icons.qrc">:/22x22/gamma.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QSlider" name="gammaSlider">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="gammaText">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(230,230,230);</string>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AMColorPickerButton</class>
   <extends>QToolButton</extends>
   <header>ui/AMColorPickerButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
